### PR TITLE
Golf 4451 fix resto sync

### PR DESCRIPTION
**Pull request by lspdMatt**
- removes list endpoint for "Product Group Product"
- adds find endpoint for "Inventory Product"

### CODE REVIEW

File: lib/lightspeed_restaurant/inventory_product.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'lightspeed_restaurant/base'
+require 'lightspeed_restaurant/operations/list'
```
Comment (olimart) [inline lib/lightspeed_restaurant/inventory_product.rb:4]: needed?

File: lib/lightspeed_restaurant.rb [hunk 2]
```diff
@@ -19,6 +18,7 @@
 require 'lightspeed_restaurant/payment_type'
 require 'lightspeed_restaurant/product_group'
 require 'lightspeed_restaurant/product_group_product'
+require 'lightspeed_restaurant/inventory_product'
```
Comment (olimart) [inline lib/lightspeed_restaurant.rb:21]: can you sort please.

**Review by olimart — APPROVED** [review]
Back to single quotes world ;) 
Looking good. Thanks

File: lib/lightspeed_restaurant/inventory_product.rb (hunk 1 above)
Comment (lspdMatt) [reply lib/lightspeed_restaurant/inventory_product.rb]: should have been find, but I guess it was already required somewhere else. Updated it to `find`

File: lib/lightspeed_restaurant/company.rb [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'lightspeed_restaurant/base'
+require 'lightspeed_restaurant/operations/find'
```
Comment (lspdMatt) [inline lib/lightspeed_restaurant/company.rb:4]: after reordering the requires, realized this was missing

**Pull request merged**